src/osw_config_keys.cpp

```diff
@@ -95,6 +95,7 @@ OswConfigKeyInt distPerDay("f2", "Fitness", "Distance per day", "> 0!", STEPS_PE
 OswConfigKeyInt kcalPerDay("f3", "Fitness", "kcalorie per day", "> 0!", STEPS_PER_DAY);
 OswConfigKeyBool stepsHistoryClear("o", "Fitness", "Clear historical days", "In case the watch did not run for multiple days, these will be cleared. Can make problems if time is lost during sleep.", STEPS_HISTORY_CLEAR);
 #endif
+OswConfigKeyString weather("wtr","weather","encoded weather","","X");
```

Comment:
I think you (@Lorenzosciacca ) should not use the configuration to cache such values. For these, please still use the classic `Preferences.h` approach by introducing a namespace for yourself. This is because the UI is not happy if we start shipping arbitrary long values and may even crash it. Yay!

Reply:
for Preferences.h you mean [this](https://github.com/espressif/arduino-esp32/blob/master/libraries/Preferences/src/Preferences.cpp) ? So using nvs without the wrapper?

Reply:
Yes, try to use `Preferences.h` without the OswConfig-wrapper :)